simple-auth/src/main/java/com/fga/example/config/OpenFgaClientTupleKeyDeserializer.java

```diff
@@ -0,0 +1,34 @@
+package com.fga.example.config;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import dev.openfga.sdk.api.client.model.ClientRelationshipCondition;
+import dev.openfga.sdk.api.client.model.ClientTupleKey;
+
+import java.io.IOException;
+
+public class OpenFgaClientTupleKeyDeserializer extends JsonDeserializer<ClientTupleKey> {
```

Comment:
I think I forgot to mark this down somewhere else.

Jackson wasn't having a fun time with the `ClientTupleKey`, so I just put in a custom Deserializer for the class instead of digging into it too much.